common/templates/context.go

```diff
@@ -53,6 +53,7 @@ var (
 		"print":       withOutputLimit(fmt.Sprint, MaxStringLength),
 		"println":     withOutputLimit(fmt.Sprintln, MaxStringLength),
 		"printf":      withOutputLimitF(fmt.Sprintf, MaxStringLength),
+		"deSpoof":     common.DeSpoof,
```

Comment:
Out of curiosity, have you tested this? `common.DeSpoof` has two return values (the second of which isn't the `error` type), which I don't believe is supported.

Reply:
I sadly have not tested it yet (hold your laughter, please)
But, if it doesn't support returning str & bool then I can probably make a quick fix for that

Reply:
Well, I tested the func itself, but I still struggle to get a bare bones selfhost functional lol

Reply:
Yeah from what I see at line 312, I think it might not support returning str and bool, I can just move the logic for checking if message was modified into CheckMessage though.